### PR TITLE
Form: Allow default values updates

### PR DESCRIPTION
**Pull request by dprokop**

### CODE REVIEW

File: packages/grafana-ui/src/components/Forms/Form.story.tsx [hunk 1]
```diff
@@ -122,17 +122,39 @@ export const basic = () => {
...
+        path: 'Nested path default value',
+      },
+      radio: 'option2',
+      select: 'option1',
+      switch: true,
+    },
+    {
+      name: 'John Waters',
+      nested: {
+        path: 'Some shit',
```
Comment (dprokop) [inline packages/grafana-ui/src/components/Forms/Form.story.tsx]:
```suggestion
        path: 'Nested path default value updated',
```

Comment (tskarhed) [reply packages/grafana-ui/src/components/Forms/Form.story.tsx]: Lol

Comment (dprokop) [reply packages/grafana-ui/src/components/Forms/Form.story.tsx]: should have force pushed instead :D

**Review by tskarhed — APPROVED** [review]
Coolers as we say in Sweden!

**Pull request merged**